### ISSUE TITLE
NTR: Neutralizing anti-rabies virus antibodies in blood (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8744]
New term request Neutralizing anti-rabies virus antibodies in blood (samant_110131003343):

**Comment by pnrobinson** [comment]
@MickeySegal


Adding new term: 
- ID: HP:6000929
- label: Neutralizing anti-rabies virus antibody positivity
- parents: [Positive bloodstream pathogen-specific antibody positivity (HP:0430051)](https://hpo.jax.org/app/browse/term/HP:0430051)
- pmid: PMID:38075404
- def: The presence of antibodies in the blood circulation that act against the rabies virus function by blocking the binding of the glycoprotein to a receptor or preventing the membrane fusion process.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8744
- synonyms:
